cmd/root.go

```diff
@@ -15,6 +15,7 @@ type rootFlags struct {
 	verbose    bool
 	allPods    bool
 	json       bool
+	config     string
```

Comment:
I think it should be named something other than config, given that the names of the YAML files we anticipate to audit with this are called manifests.